gst/nnstreamer/tensor_query/nnstreamer_edge_common.h

```diff
@@ -83,21 +89,11 @@ typedef struct {
 #define nns_edge_logd g_debug
 #define nns_edge_logf g_error
 
-/**
- * @brief Validate edge handle.
- */
-bool nns_edge_handle_is_valid (nns_edge_h edge_h);
-
 /**
  * @brief Check network connection.
  */
 bool nns_edge_is_connected (nns_edge_h edge_h);
 
-/**
- * @brief Validate data handle.
- */
-bool nns_edge_data_is_valid (nns_edge_data_h data_h);
-
 /**
  * @brief Create nnstreamer edge event.
  */
```

Comment:
Rough suggestion: Let's rename ```nns_edge_event_destroy``` as ```_nns_edge_event_destroy```, and
```
#define nns_edge_event_destroy(event_h)  do { _nns_edge_event_destroy(event_h); event_h=NULL; } while (0)
```
Do you prefer it?

Reply:
need to handle return code of nns_edge_event_destroy()